packages/nuxt3-module/plugin.ts

```diff
@@ -26,12 +26,21 @@ declare module "vue" {
 
 export default defineNuxtPlugin((NuxtApp) => {
   const runtimeConfig = useRuntimeConfig();
-  const shopwareEndpoint =
+  const shopwareEndpointCSR =
     runtimeConfig.public?.shopware?.endpoint ??
     runtimeConfig.public?.shopware?.shopwareEndpoint;
+
+  const shopwareEndpointSSR =
+    runtimeConfig.shopware?.endpoint ?? shopwareEndpointCSR;
```

Comment:
maybe we could add some warning in the build type in `src/index.ts` that in SSR the endpoint defined for CSR will be used as a fallback - or other way around.

Reply:
I'm not convinced because ...
- This SSR endpoint should be used only for a very specific occasion, so we should avoid logging unnecessary information. The developer can easily debug it
- Plugins are triggered by the request, so the console will be spammed. 
- CRS requests never reach the SSR endpoint

Reply:
I was only mentioning to log it in the build time in the module, not a plugin - so no spamming here. we should better inform the developer what endpoint/configuration is being used, because some of them will never reach our repository to find the specific line of code - that's just my opinion 🙌🏼

Reply:
Ah got it, done